src/patients/related-persons/RelatedPersonTab.tsx

```diff
@@ -87,14 +90,18 @@ const RelatedPersonTab = (props: Props) => {
       <div className="row">
         <div className="col-md-12">
           <Panel title={t('patient.relatedPersons.label')} color="primary" collapsible>
-            {relatedPersons ? (
-              <List>
-                {relatedPersons.map((r) => (
-                  <ListItem key={r.id}>{r.fullName}</ListItem>
-                ))}
-              </List>
-            ) : (
-              <h1>Loading...</h1>
+            {dataFetched ? (
```

Comment:
We'll need tests covering this functionality: 

* test for loading bar while loading
* test for the message when there are no related persons